libs/template/labs/src/i18n/translations/en.ts

```diff
@@ -39,6 +39,8 @@ const user = {
 const picking = {
   'picking.product-card.of-<count>-items':
     'Of {count, plural, one {{count} item} other {{count} items}}',
+  'picking.filter.<count>-open-pick-lists':
+    '{count, plural, one {{count} Open Pick List} other {{count} Open Pick Lists}}',
```

Comment:
This can be expressed a bit more optimally:
```suggestion
    '{count} Open Pick {count, plural, one {List} other {Lists}}',
```